lib/writeGeometryToGltf.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+var writeAccessor = require('./writeAccessor');
+
+module.exports = writeGeometryToGltf;
+
+function writeGeometryToGltf(gltf, primitive, geometry) {
+    var positionId = primitive.attributes.POSITION;
+    var positionAccessor = gltf.accessors[positionId];
+
+    var indicesId = primitive.indices;
+    var indicesAccessor = gltf.accessors[indicesId];
+
+    var values = geometry.attributes.position.values;
+
+    writeAccessor(gltf, positionAccessor, values);
+    writeAccessor(gltf, indicesAccessor, geometry.indices);
+}
```

Comment:
Can you also include support for normals here since I will be using it? First check that the primitive has a NORMAL attribute though.